pkg/util/extract_forwarded.go

```diff
@@ -0,0 +1,53 @@
+package util
+
+import (
+	"context"
+
+	"google.golang.org/grpc/metadata"
+)
+
+// ipAddressesKey is key for the GRPC metadata where the IP addresses are stored
+const ipAddressesKey = "x-forwarded-for"
+
+// GetSourceFromOutgoingCtx extracts the source field from the GRPC context
```

Comment:
This is a generic `util` package. The function name `GetSourceFromOutgoingCtx()` looks a big vague to me. Maybe `GetSourceIPsFromOutgoingCtx` could me more clear? If so, other functions in this file should be renamed accordingly.

Reply:
That naming is more definitely more consistent. It's changed.